File: Logic-1/squirrelPlay/SquirrelPlay.java
package squirrelPlay;

public class SquirrelPlay {
    //Retorne true se "temp" estiver no intervalo de 60 a 90, inclusive, a menos que "isSummer" seja true, nesse caso o
    //o limite máximo é 100, inclusive.

    public boolean squirrelPlay(int temp, boolean isSummer){
        if (isSummer && temp <= 100 && temp >= 60) return true;
        else return (temp >= 60 && temp <= 90);
    }
}

File: Python/Logic-1/squirrel_play.py
"""
Dado um número inteiro "temp" e um tipo booleano "is_summer", retorne true caso
"temp" estiver entre 60 e 90, inclusive, a menos que "is_summer" seja true aí
o limite superior máximo passa a ser 100.
"""


def squirrel_play(temp, is_summer):
    if temp >= 60:
        if temp > 90:
            return temp <= 100 and is_summer
        return True
    return False


print(squirrel_play(95, True))
